### PR TITLE
Updates the field with the new Webots RoboCup field

### PR DESCRIPTION
**Pull request by ysims**
Cyberbotics updated the RoboCup field a lil bit ago. This updates our stuff with their stuff.

### CODE REVIEW

File: worlds/adult.wbt [hunk 1]
```diff
@@ -1,74 +1,108 @@
...
+#VRML_SIM R2021b utf8
 WorldInfo {
   info [
     "Description: official soccer simulation for the 2021 Robocup Virtual Humanoid League (adult size)"
-    "Version 0.1"
+    "Version 0.2"
   ]
   title "Robocup V-HL Adult"
-  basicTimeStep 16
+  basicTimeStep 8
```
Comment (Lachourt) [inline worlds/adult.wbt:8]: What's the benefit of changing the timestep? Or is that just the standard for robocup?

**Review by Lachourt — COMMENTED** [review]
Is there a reason we're still keeping the adult world up to date if we're only entering the kid league? Or have I misunderstood something

File: worlds/adult.wbt (hunk 1 above)
Comment (ysims) [reply worlds/adult.wbt:8]: Physics will update faster. I think it is the standard, or at least the conditions we'll be playing under (I don't think we're meant to get anymore field changes). It's not that necessary but may be useful if we want to play under their conditions.

**Comment by ysims** [comment]
> Is there a reason we're still keeping the adult world up to date if we're only entering the kid league? Or have I misunderstood something

I've been thinking about this. I'm tempted to keep it because we do want to compete in the future in adult size, but the field might be different by then anyway...

File: worlds/adult.wbt (hunk 1 above)
Comment (Lachourt) [reply worlds/adult.wbt:8]: Fair enough, a smaller timestep will be great for vision data collection anyway so

**Comment by Lachourt** [comment]
LGTM but I can't approve 👍

**Review by Lachourt — APPROVED** [review]
Wait no I can approve 😀

**Pull request merged**